definitions/m5/m5.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: m5-release-pipeline
+  namespace: managed
+spec:
+  params:
+    - name: mapping-file
+      type: string
+      description: mapping to apply to snapshot
+  tasks:
+    - name: get-mapping-file
+      taskRef:
+        name: get-mapping-file
+      params:
+        - name: mapping-file
+          value: "$(params.mapping-file)"
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-mapping-file
+  namespace: managed
+spec:
+  params:
+    - name: mapping-file
+      type: string
+      description: the file that defines the mapping to apply to the snapshot
+  results:
+    - name: mapping
+      description: The retrieved mapping file as a string
+  steps:
+    - name: get-mapping-file
+      image: quay.io/fedora/fedora:36
```

Comment:
We don't really need Fedora. Maybe a minimal distro like `alpine` would be better. Although I like that you got it from quay. This avoids Docker pull limitations.